incubator/catalog-filesystem.spec/src/main/scripts/io/aklivity/zilla/specs/catalog/filesystem/config/event.yaml

```diff
@@ -36,6 +36,7 @@ bindings:
     type: test
     kind: server
     options:
-      catalogs:
-        - catalog0
+      catalog:
+        catalog0:
+          - id: 1
```

Comment:
It would seem more readable and descriptive of the intent to put `subject: not-subject1` instead of `id: 1` here, agree?